task-launcher/src/tasks/shared/trials/afcStimulus.js

```diff
@@ -75,10 +76,55 @@ function getStimulus(trialType) {
   }
 }
 
+const getPromptTemplate = (prompt, mediaSrc, mediaAlt, stimText, equalSizeStim) => {
```

Comment:
Nice, I like this. Could you add JS Doc annotations for this function?

Reply:
thanks for your thorough review, Phil! @asengupta3 , we definitely want larger (more screen-filling) images for the picture-based buttons (TROG, MR, vocab). And yes, wonder where the instruction trials have gone (they are part of the corpora: https://docs.google.com/spreadsheets/d/1MlU4eOd45XVMg7HrnTDGZ3rv1cfNjvjpdc8e_edQqQk/edit?usp=sharing )